code/modules/mob/living/simple_animal/abnormality/aleph/white_night.dm

```diff
@@ -73,6 +76,17 @@ GLOBAL_LIST_EMPTY(apostles)
 				G.forceMove(T)
 			revive_humans()
 
+/mob/living/simple_animal/hostile/abnormality/white_night/death(gibbed)
+	if (sinned)
+		return ..()
+	for(var/mob/living/carbon/human/heretic in heretics)
+		if(heretic.stat == DEAD || !heretic.ckey)
+			continue
+		heretic.Apply_Gift(new /datum/ego_gifts/blessing)
+		playsound(src, 'sound/abnormalities/whitenight/apostle_bell.ogg', 50)
```

Comment:
This plays the sound at WN's location.
Additionally, you might want to use the local proc instead, so only the player hears it and not everyone around them.